addOns/custompayloads/src/main/java/org/zaproxy/zap/extension/custompayloads/CustomMultiplePayloadDialog.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2021 The ZAP Development Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.zap.extension.custompayloads;
+
+import java.awt.Window;
+import java.io.File;
+import javax.swing.JButton;
+import javax.swing.JFileChooser;
+import org.parosproxy.paros.Constant;
+import org.parosproxy.paros.view.View;
+import org.zaproxy.zap.utils.DisplayUtils;
+
+public class CustomMultiplePayloadDialog extends AbstractColumnDialog<CustomPayload> {
+
+    private static final long serialVersionUID = 1L;
+    private static final String SELECT_FILE_BUTTON =
+            Constant.messages.getString(
+                    "custompayloads.options.dialog.addMultiplePayload.selectFile.button.name");
+    private JButton fileButton;
+    private File multiplePayload;
+
+    public CustomMultiplePayloadDialog(Window owner, String title, CustomPayload payload) {
+        super(
+                owner,
+                title,
```

Comment:
The title does not need to be customisable.

Reply:
@bettercalln1ck to address this comment `"custompayloads.options.dialog.title"` could just be passed to super here, instead of taking `String title` as a param for the constructor.

Reply:
That's not the correct key though.

Reply:
Sorry, yes my mistake. I guess a proper title key value pair should be defined (in Messages.properties) and used.